### PR TITLE
Moyo header

### PR DESCRIPTION
**Pull request by Dimoshua**
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,96 @@
...
+body {
+  margin: 0;
+  --main-color: #00acdc;
+}
+
+.container {
+  padding: 0 50px;
+
+  display: flex;
+  flex-direction: row;
```
Comment (SanyaBratashchuk) [inline src/style.css:28]:
```suggestion
```
Default value

File: src/style.css [hunk 2]
```diff
@@ -1 +1,96 @@
...
+  position: relative;
+  width: 40px;
+  height: 40px;
+  background-image: url("./images/logo.png");
+  background-repeat: no-repeat;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
```
Comment (SanyaBratashchuk) [inline src/style.css:46]:
```suggestion
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,96 @@
...
+
+.nav__list:first-child::after {
+  content: "";
+  position: absolute;
+  background: var(--main-color);
+  height: 4px;
+  width: 37px;
+  margin: 56px auto 0;
+  border-radius: 8px;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:67]: move it to `.is-active::after`

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (Dimoshua) [reply src/style.css:67]: If they change like this, then the tests will not pass. Tests do not see the underlined line

Comment (Dimoshua) [reply src/style.css:67]: I solved the problem by inserting the is-active class twice:

<ul class="nav__list is-active">
            <a class="box is-active" href="#apple">Apple</a>
          </ul>

Comment (Dimoshua) [reply src/style.css:67]: I solved the problem by inserting the is-active class twice in *.html

**Pull request closed without merging**